<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="VICKY" name="Mobile Automation Test on Pixel 2" tests="3" failures="3" timestamp="22 Dec 2020 04:31:15 GMT" time="64.247" errors="0">
  <testcase name="invalidUsernameTest" time="8.862" classname="com.TestCases.LoginTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [null] but found [Username and password do not match any user in this service.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [null] but found [Username and password do not match any user in this service.]
	at com.TestCases.LoginTest.invalidUsernameTest(LoginTest.java:57)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- invalidUsernameTest -->
  <testcase name="invalidPasswordTest" time="5.122" classname="com.TestCases.LoginTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [null] but found [Username and password do not match any user in this service.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [null] but found [Username and password do not match any user in this service.]
	at com.TestCases.LoginTest.invalidPasswordTest(LoginTest.java:69)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- invalidPasswordTest -->
  <testcase name="validUsernameTest" time="8.29" classname="com.TestCases.LoginTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [null] but found [PRODUCTS]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [null] but found [PRODUCTS]
	at com.TestCases.LoginTest.validUsernameTest(LoginTest.java:81)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- validUsernameTest -->
</testsuite> <!-- Mobile Automation Test on Pixel 2 -->
